tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -1013,14 +1015,14 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                                    'FP32 model weights and biases are compressed to FP16. '
                                    'All intermediate data is kept in original precision.',
                               choices=["FP16", "FP32", "half", "float"],
-                              default='float',
+                              default=None,
```

Comment:
also, please return `float`

Reply:
removed `None` returned default value, but `FP16` instead of float